src/common/multi_file_reader.cpp

```diff
@@ -362,7 +424,10 @@ HivePartitioningIndex HivePartitioningIndex::Deserialize(Deserializer &source) {
 void MultiFileReaderOptions::AddBatchInfo(BindInfo &bind_info) const {
 	bind_info.InsertOption("filename", Value::BOOLEAN(filename));
 	bind_info.InsertOption("hive_partitioning", Value::BOOLEAN(hive_partitioning));
+	bind_info.InsertOption("auto_detect_hive_partitioning", Value::BOOLEAN(auto_detect_hive_partitioning));
 	bind_info.InsertOption("union_by_name", Value::BOOLEAN(union_by_name));
+	bind_info.InsertOption("hive_types_autocast", Value::BOOLEAN(hive_types_autocast));
+	// hive_types_schema?
```

Comment:
is there an option missing here? or is this a left-over comment?

Reply:
Ah, I'm not sure if I need to add the schema to this function, and if so, if it should be as a struct or map